Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/transform/AuditTransforms.java

```diff
@@ -112,7 +116,7 @@ public final LogEventRequestType transformRequestToAuditMsg(T request, Assertion
         return buildLogEventRequestType(auditMsg, direction, getMessageCommunityId(assertion, target, isRequesting),
             serviceName, assertion, auditMsg.getEventIdentification().getEventID().getDisplayName(),
             auditMsg.getEventIdentification().getEventOutcomeIndicator(),
-            auditMsg.getEventIdentification().getEventDateTime(), getUserId(auditMsg.getActiveParticipant()));
+            auditMsg.getEventIdentification().getEventDateTime(), getRequestorUserId(auditMsg,assertion));
```

Comment:
I guess it is easy to check on null on whole object rather than its member.  We can pass getActiveParticipant object into the method but that assume the auditMsg is not null.